apps/synapse-oauth-signin/src/LogoutComponent.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { useEffect } from 'react'
+import { SynapseClient, useSynapseContext } from 'synapse-react-client'
+
+export function LogoutComponent() {
```

Comment:
Existing component, just pulled out into its own file. I don't think it's used, but I'm trying not to make many functional changes in this PR.